core/include/moveit/task_constructor/stage_p.h

```diff
@@ -116,7 +114,7 @@ class StagePrivate
 	/// enforce only one parent exists
 	inline bool setParent(ContainerBase* parent) {
 		if (parent_) {
-			ROS_ERROR_STREAM("Tried to add stage '" << name() << "' to two parents");
```

Comment:
For consistency, you should also remove the return value. Throwing _and_ returning false doesn't make sense.